src/core/sys/windows/winnt.d

```diff
@@ -3316,13 +3325,28 @@ struct IMAGE_LOAD_CONFIG_DIRECTORY64 {
 }
 alias IMAGE_LOAD_CONFIG_DIRECTORY64* PIMAGE_LOAD_CONFIG_DIRECTORY64;
 
-struct IMAGE_RUNTIME_FUNCTION_ENTRY {
+version (Win64) {
+    alias IMAGE_LOAD_CONFIG_DIRECTORY64 IMAGE_LOAD_CONFIG_DIRECTORY;
+} else {
+    alias IMAGE_LOAD_CONFIG_DIRECTORY32 IMAGE_LOAD_CONFIG_DIRECTORY;
+}
+alias IMAGE_LOAD_CONFIG_DIRECTORY* PIMAGE_LOAD_CONFIG_DIRECTORY;
+
+struct IMAGE_ALPHA_RUNTIME_FUNCTION_ENTRY { // for Alpha...need?
```

Comment:
I think we can just replace it with the correct definition. Having an Alpha declaration, but no Alpha64 or ARM declaration seems pointless.